generators/generator-base.js

```diff
@@ -1531,6 +1531,29 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
     return this.getExistingEntities().map(entity => entity.name);
   }
 
+  /**
+   * Read entity json from config folder.
+   * @param {string} entityName - Entity name
+   * @return {object} entity definition
+   */
+  readEntityJson(entityName) {
```

Comment:
the function's name is pretty clear, no need for comments

Reply:
if you feel the need to comment something, then something might not be clear
if it's the case for you here, then perhaps being a bit more verbose may be what you want